src/reducers/books.js

```diff
@@ -0,0 +1,32 @@
+const CREATE_BOOK = 'CREATE_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const initialState = {
+  books: [
+    {
+      id: 1,
+      title: 'See, Touch, Feel',
+      category: 'Kids',
+
+    },
+    {
+      id: 2,
+      title: 'The Sum of Us',
+      category: 'History',
+    },
+  ],
+};
```

Comment:
- [ ] Before you merge, move `initialState` to the `src/index.js`  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) from pathwright)